src/nvim/api/ui_events.in.h

```diff
@@ -58,7 +58,8 @@ void set_title(String title)
   FUNC_API_SINCE(3);
 void set_icon(String icon)
   FUNC_API_SINCE(3);
-
+void highlight_info_set(Array highlights_changed)
+  FUNC_API_SINCE(3);
```

Comment:
Why not include the translated colors in the mode_info dict? We should avoid unnecessary indirections.